server/db/models/customer.js

```diff
@@ -5,6 +5,7 @@ const db = require('../db')
 const Customer = db.define('customer', {
   name: {
     type: Sequelize.STRING,
+    unique: true,
```

Comment:
Customer name doesnt need to be unique. Two jordans should be able to create an account.